### PR TITLE
profile section update

### PR DESCRIPTION
**Pull request by s-shuja-rehman**

### CODE REVIEW

File: src/components/ProfileWrapper/Friends/Follower.jsx [hunk 1]
```diff
@@ -0,0 +1,238 @@
+// import "swiper/css/swiper.min.css";
+
+import { useState } from "react";
+
+export const Follower = () => {
+    const [iconTab, setIconTab] = useState({
+        grid: true,
+        column: false,
+      });
```
Comment (i-abdul-raheem) [inline src/components/ProfileWrapper/Friends/Follower.jsx]: change this state to 
const [isgrid, setIsGrid] = useState(true);

File: src/components/ProfileWrapper/Friends/Following.jsx [hunk 2]
```diff
@@ -0,0 +1,221 @@
+// import "swiper/css/swiper.min.css";
+
+import { useState } from "react";
+
+export const Following = () => {
+  const [iconTab, setIconTab] = useState({
+    grid: true,
```
Comment (i-abdul-raheem) [inline src/components/ProfileWrapper/Friends/Following.jsx]: Same as above

File: src/components/ProfileWrapper/MyPost/MyPost.jsx [hunk 3]
```diff
@@ -0,0 +1,221 @@
+import { useState } from "react";
+import { NavLink } from "react-router-dom";
+
+export const MyPost = () => {
+  const [iconTab, setIconTab] = useState({
+    grid: true,
```
Comment (i-abdul-raheem) [inline src/components/ProfileWrapper/MyPost/MyPost.jsx]: Same as above

File: src/components/ProfileWrapper/MyPost/MyPost.jsx [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+                  </a>
+                ))}
+              </div>
+            ))}
+          </div>
+        )}
+        {/* {profileData.length > 0 ? (
+          profileData.map((tab, index) => (
+            <div
+              className={`tab-pane fade ${index === 0 ? "show active" : ""}`}
```
Comment (i-abdul-raheem) [inline src/components/ProfileWrapper/MyPost/MyPost.jsx]: Remove commented code

**Review by i-abdul-raheem — CHANGES_REQUESTED** [review]

File: src/components/ProfileWrapper/Friends/Follower.jsx (hunk 1 above)
Comment (s-shuja-rehman) [reply src/components/ProfileWrapper/Friends/Follower.jsx]: done

File: src/components/ProfileWrapper/MyPost/MyPost.jsx (hunk 3 above)
Comment (s-shuja-rehman) [reply src/components/ProfileWrapper/MyPost/MyPost.jsx]: done

File: src/components/ProfileWrapper/MyPost/MyPost.jsx (hunk 4 above)
Comment (s-shuja-rehman) [reply src/components/ProfileWrapper/MyPost/MyPost.jsx]: done

File: src/components/ProfileWrapper/Friends/Following.jsx (hunk 2 above)
Comment (s-shuja-rehman) [reply src/components/ProfileWrapper/Friends/Following.jsx]: done

**Review by i-abdul-raheem — APPROVED** [review]

**Pull request merged**